src/gui/guiFormSpecMenu.cpp

```diff
@@ -2794,6 +2795,11 @@ void GUIFormSpecMenu::parseModel(parserData *data, const std::string &element)
 	e->enableContinuousRotation(inf_rotation);
 	e->enableMouseControl(mousectrl);
 
+	const s32 frame_loop_begin = stoi(frame_loop[0]);
+	const s32 frame_loop_end = frame_loop.size() == 1 ? 0x7FFFFFFF : stoi(frame_loop[1]);
```

Comment:
This argument is optional, but `lua_api.txt` doesn't say as such.  Either make it required or document that it is optional.